caching-service-package/src/main/resources/bin/start.sh

```diff
@@ -269,6 +269,7 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${CACHING_CODE} ${JAVA_BIN_DIR}java \
   -Dserver.ssl.trustStoreType="${truststore_type}" \
   -Dserver.ssl.trustStorePassword="${truststore_pass}" \
   -Djava.protocol.handler.pkgs=com.ibm.crypto.provider \
+  -Djava.net.preferIPv4Stack=true \
```

Comment:
what is for?

Reply:
There are some warnings coming from infinispan about defaulting to IPv6 but trying to connect to an IPv4 cluster

Reply:
I see, ok, thanks for the clarification.